[LYNX-90] enable working example with download

lecstor opened the pull request:
- implement `npm run example:export`
- fix main export Babel6 issue
- add function to enable downloading export files

lecstor committed:
[LYNX-90] enable working example with download

- implement `npm run example:export`
- fix main export Babel6 issue

koresar commented on src/api.js:
```diff
@@ -1,4 +1,5 @@
 import rp from 'request-promise';
+import request from 'request';
```
Why we need `request` here in addition to `request-promise`?

lecstor replied:
the download method I added streams the download to the file. `request-promise` recommends not using it for streaming but rather using plain `request`.

koresar replied:
:+1:

lecstor committed:
[LYNX-90] download fixes

- don't use download return as a promise.. it's not
- decode filename extracted from uri
- add password example, update end points

johndagostino commented:
made some changes, merging
